build/CMakeCache.txt

```diff
@@ -0,0 +1,518 @@
+//ADVANCED property for variable: CMAKE_RANLIB
+CMAKE_RANLIB-ADVANCED:INTERNAL=1
+//Path to CMake installation.
+CMAKE_ROOT:INTERNAL=/usr/share/cmake-2.8
+//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS
+CMAKE_SHARED_LINKER_FLAGS-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_DEBUG
+CMAKE_SHARED_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL
+CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_RELEASE
+CMAKE_SHARED_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO
+CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_SKIP_INSTALL_RPATH
+CMAKE_SKIP_INSTALL_RPATH-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_SKIP_RPATH
+CMAKE_SKIP_RPATH-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS
+CMAKE_STATIC_LINKER_FLAGS-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_DEBUG
+CMAKE_STATIC_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL
+CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_RELEASE
+CMAKE_STATIC_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO
+CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_STRIP
+CMAKE_STRIP-ADVANCED:INTERNAL=1
+//uname command
+CMAKE_UNAME:INTERNAL=/bin/uname
+//ADVANCED property for variable: CMAKE_USE_RELATIVE_PATHS
+CMAKE_USE_RELATIVE_PATHS-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: CMAKE_VERBOSE_MAKEFILE
+CMAKE_VERBOSE_MAKEFILE-ADVANCED:INTERNAL=1
+//List of iCub libraries
+ICUB_LIBRARIES:INTERNAL=iCubIDLClients;iCubDev;iCubVis;logpolar;boostMIL;ctrlLib;skinDynLib;iKin;iDyn;learningMachine;perceptiveModels;actionPrimitives;cartesiancontrollerserver;cartesiancontrollerclient;gazecontrollerclient;debugInterfaceClient;icubmod;d4c;optimization;lib0;lib1;sharksWithLasers
+//List of iCub linker options
+ICUB_LINK_FLAGS:INTERNAL=
+//iCub cmake scripts directory
+ICUB_MODULE_PATH:INTERNAL=/usr/local/share/iCub/cmake
+//ADVANCED property for variable: YCM_USE_3RDPARTY
+YCM_USE_3RDPARTY-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: YCM_USE_CMAKE_3_0
+YCM_USE_CMAKE_3_0-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: YCM_USE_CMAKE_NEXT
+YCM_USE_CMAKE_NEXT-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: YCM_USE_CMAKE_PROPOSED
+YCM_USE_CMAKE_PROPOSED-ADVANCED:INTERNAL=1
+//ADVANCED property for variable: YCM_USE_DEPRECATED
+YCM_USE_DEPRECATED-ADVANCED:INTERNAL=1
+//CMAKE_INSTALL_PREFIX during last run
+_GNUInstallDirs_LAST_CMAKE_INSTALL_PREFIX:INTERNAL=/usr/local
+
```

Comment:
@naveenoid check also all this files that you committed in the `build` directory.